client/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
These files in the client should be removed because we dont use them.

Reply:
WIP.